### PR TITLE
GraphQL Basics

### PR DESCRIPTION
**Pull request by ArtemKhamzin**

### CODE REVIEW

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  end
+
+  def show
+  end
+
+  def new
+  end
+
+  def create
+    task = Task.new(task_params)
```
Comment (DmitryBarskov) [inline app/controllers/tasks_controller.rb:15]: You should this code, it doesn't work

File: app/graphql/mutations/create_project.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+module Mutations
+    class CreateProject < BaseMutation
```
Comment (DmitryBarskov) [inline app/graphql/mutations/create_project.rb:2]: You should implement all CRUD operations for this task

File: app/views/tasks/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<%= form_with model: task do |form| %>
```
Comment (DmitryBarskov) [inline app/views/tasks/_form.html.erb:1]: Why do you add simple_form but don't use it?

**Review by DmitryBarskov — CHANGES_REQUESTED** [review]

File: app/views/tasks/_form.html.erb (hunk 3 above)
Comment (KilinMikhail) [reply app/views/tasks/_form.html.erb:1]: This task should not contain this code, so we can ignore this problem

File: app/controllers/tasks_controller.rb (hunk 1 above)
Comment (KilinMikhail) [reply app/controllers/tasks_controller.rb:15]: This task should not contain this code, so we can ignore this problem. We discussed it in the lesson

**Pull request closed without merging**